packages/utilities/psammead-storybook-helpers/src/decorators.js

```diff
@@ -0,0 +1,7 @@
+import inputProvider from './input-provider';
+
+/* eslint-disable import/prefer-default-export */
+export const dirDecorator = storyFn => {
```

Comment:
Just like how we have https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-storybook-helpers/src/index.stories.jsx which shows off the input provider, can you make a simple example of the dirDecorator please in a real stories.jsx file